README.md

```diff
@@ -0,0 +1,21 @@
+# It's ALIVE!
+It's ALIVE! is a single-player VR experience in which you play the Doctor's Assistant. Your goal: to bring the doctor's creation to life!
+
```

Comment:
I'd also recommend having a picture here! And given the other contents of this branch, you can write more of the Readme after in another branch/PR.

Reply:
Oh great idea!

Reply:
Added a TODO note so I remember to put the picture in!